src/plugins/aws/ssm/index.ts

```diff
@@ -225,19 +228,21 @@ function spawnChildProcess(
     env: {
       ...process.env,
       ...credential,
+      ...(sshAgentEnv || {}),
```

Comment:
I thought using the spread operator `...` on an undefined value just did nothing 🤔

Reply:
It compiles here but that's surprising to me... 
I tried with a toy example:
![image](https://github.com/p0-security/p0cli/assets/5836460/be82d9ff-f5a1-4a75-8b0c-446183f5657b)